tests/Resources/ODS.robot

```diff
@@ -6,6 +6,9 @@ Resource            ./Page/LoginPage.robot
 Resource            ./Page/ODH/ODHDashboard/ODHDashboard.resource
 Resource            ./Page/ODH/JupyterHub/ODHJupyterhub.resource
 Resource            ./Page/ODH/Prometheus/Prometheus.resource
+Resource            ../../tasks/Resources/RHODS_OLM/uninstall/uninstall.robot
```

Comment:
@pablofelix If tests start using keywords in RHODS_OLM I think it will make sense to have a common Resource forder for tests and tasks.

Proposal:
```
Rename ods-ci/tests to ods/src and have this 3 subfolders:
ods-ci/src/Tests/
ods-ci/src/Tasks/
ods-ci/src/Resources/  (common resource folder for tests and tasks)
```

It doesn't need to be done in this PR but I think we should talk more about this. 

@lugi0 @bdattoma @tarukumar @aloganat I would like to know that do you think about this

Reply:
we could discuss it in a dedicate meeting and considering pros and cons of both the approaches

Reply:
+1 to reorganizing the folders. The PR doing that will likely also need to update all of the imports we currently have, as the paths will change.